### PR TITLE
Update glossary.md

### PR DESCRIPTION
**Pull request by spiral6**
Added and fixed links, fixed typos, made glossary terms more readable

### CODE REVIEW

File: docs/extras/glossary.md [hunk 1]
```diff
@@ -6,33 +6,33 @@ This section is dedicated to explaining a number of common terms that are used w
...
-- ReSwitched: A hacking team that is one of the main developers for Atmosphere.
-- RCM: A shorthand for "Recovery Mode". When talking about hacking the Switch, this commonly refers to the Recovery Mode in the Tegra X1 chip that is included in the Nintendo Switch. Can be entered by holding down the Tegra X1 home button. This button is not the same as the home button on the joycons. Ways of pressing this button can be found [here](/user_guide/entering_rcm/).
-- TegraRCMGUI/fusee-interface-gtk: Software used to run the fusee-gelee exploit on the Switch.
-- Tegra X1: A chip made by Nvidia that is used in the Nintendo Switch.
-- Trinket: A tiny chip that is soldered on the Switch motherboard to automatically send a payload if it is in RCM.
-- TrustZone: The highest security level on the Switch.
-- XCI/NSP: Format used to dump games to. XCI is used for gamecard dumps, while NSP is for dumping digital titles.
+- **90DNS**: DNS that blocks system updates on the Nintendo Switch. This can be enabled by following the guide [here](switch-guide/extras/blocking_updates/).
```
Comment (Oreo639) [inline docs/extras/glossary.md:9]: Please just make this `[here](blocking_updates.md)`

File: docs/extras/glossary.md [hunk 2]
```diff
@@ -6,33 +6,33 @@ This section is dedicated to explaining a number of common terms that are used w
...
+- **boot0 and boot1**: Two partitions of the Switch NAND.
+- **CFW**: Short for custom firmware. Custom firmware permits you to completely modify how your Switch works.
+- **Deja Vu**: An unreleased exploit chain for the Nintendo Switch. The completely unpatched version of this chain will grant access to TrustZone, which effectively means being able to enter CFW. The TrustZone part of this exploit was patched in firmware version 5.0.0 and up, but normal homebrew access can still be gained up to firmware version 6.0.1.
+- **DNS**: [Domain Name System](https://en.wikipedia.org/wiki/Domain_Name_System). Basically, it's the internet's addressbook. If you visit a website, the DNS is what tells you what server the site is hosted on.
+- **Dongle**: Device you can plug in the charging port of your Switch to automatically send a payload if your Switch is in RCM.
+- **Encryption keys**: Keys used to encrypt files from the Nintendo Switch. These can be dumped by following the guide [here](switch-guide/extras/dumping_title_keys/).
```
Comment (Oreo639) [inline docs/extras/glossary.md:17]: Please make this `[here](dumping_title_keys.md)`

File: docs/extras/glossary.md [hunk 3]
```diff
@@ -6,33 +6,33 @@ This section is dedicated to explaining a number of common terms that are used w
...
+- **KIP**: Short for Kernel Initial Process. These files can be loaded in when the Switch boots in CFW and provide additional functionality.
+- **NAND**: The internal filesystem used by the Switch. Contains boot0 and boot1, as well as PRODINFO and various other partitions.
+- **nx-hbloader**: Intermediary program used to load the Homebrew Launcher from CFW developed by Switchbrew. Comes bundled with Atmosphere.
+- **PRODINFO**: A partition on the NAND of your Switch. This along with boot0 and boot1 is the only part of your Switch that can render it unbootable if improperly modified. Atmosphere backs up this partition on booting and it is included in your NAND backup.
+- **ReSwitched**: A hacking team that is one of the main developers for Atmosphere.
+- **RCM**: A shorthand for "Recovery Mode". When talking about hacking the Switch, this commonly refers to the Recovery Mode in the Tegra X1 chip that is included in the Nintendo Switch. Can be entered by holding down the Tegra X1 home button. This button is not the same as the home button on the joycons. Ways of pressing this button can be found [here](switch-guide/user_guide/emummc/entering_rcm/).
```
Comment (Oreo639) [inline docs/extras/glossary.md:30]: Please make this `[here](../user_guide/emummc/entering_rcm.md)`

**Review by Oreo639 — COMMENTED** [review]
LGTM for the most part. There's a few changes that are needed though.

File: docs/extras/glossary.md (hunk 3 above)
Comment (spiral6) [reply docs/extras/glossary.md]: Done.

File: docs/extras/glossary.md (hunk 1 above)
Comment (spiral6) [reply docs/extras/glossary.md]: Done.

File: docs/extras/glossary.md (hunk 2 above)
Comment (spiral6) [reply docs/extras/glossary.md]: Done.

**Comment by spiral6** [comment]
See above commit.

**Pull request merged**